presidential-elections/utils/model_builder.py

```diff
@@ -477,9 +479,10 @@ def build_model(
                 house_effects,
                 house_election_effects,
                 data_containers,
+                non_competing_parties
             )
 
-            concentration = pm.InverseGamma("concentration", mu=1000, sigma=100)
+            concentration = pm.InverseGamma("concentration", mu=1000, sigma=200)
```

Comment:
So the concentration parameter of the Dirichlet-Multinomial has a very intuitive interpretation: the effective total number of trials for each observation. 
Here, I took a mean of 1000 (which is roughly the sample size of any new poll), and a standard deviation of 200 to account for the variation in sample sizes. 
I changed the std because I was worried the prior was too constraining and that would bias the posterior of `concentration` (some polls do have much higher sample sizes, especially the Ipsos-Cevipof ones). It turns it doesn't change the inference that much, but I find this prior more satisfying from a domain-knowledge perspective